src/FsUnit.Xunit/CustomMatchers.fs

```diff
@@ -185,9 +185,9 @@ let instanceOfType<'a> =
 let contain expected =
     let matches(actual: obj) =
         match actual with
-        | :? (list<_>) as l -> l |> List.exists(fun i -> i = expected)
-        | :? (array<_>) as a -> a |> Array.exists(fun i -> i = expected)
-        | :? (seq<_>) as s -> s |> Seq.exists(fun i -> i = expected)
+        | :? list<_> as l -> l |> List.exists(fun i -> i = expected)
```

Comment:
do you like syntax
```fsharp
List.exists (=) expected
```
? 🙈

Reply:
That's a good suggestion. I'll apply that.